doc/developer-guides/hld/virtio-console.rst

```diff
@@ -198,3 +201,31 @@ The File backend only supports console output to a file (no input).
 #. Add the console parameter to the guest OS kernel command line::
 
       console=hvc0
+
+SOCKET
+======
+
+When virtio-console socket_type is appointed to server, device model
+will create a unix domain socket for server VM to bind and listen.
+Then other VM virtio-console socket_type is set to client, these
+VM can connect to the unix domain socket and communicate with server VM.
+
+1. Add a pci slot to the device model (``acrn-dm``) command line, adjusting
+   the ``</path/to/file.sock>`` to your use case in the VM1 configuration::
+
+      -s n,virtio-console,socket:socket_file_name=</path/to/file.sock>:server
+
+#. Add a pci slot to the device model (``acrn-dm``) command line, adjusting
+   the ``</path/to/file.sock>`` to your use case in the VM2 configuration::
+
+      -s n,virtio-console,socket:socket_file_name=</path/to/file.sock>:client
+
+#. Login to VM1, check any input to the virtual port(vportnp0, n is decided
+   by your machine)::
+
+      cat /dev/vportnp
```

Comment:
should "cat /dev/vportnp0"?

Reply:
I try to use minicom to connect the port, then transport data, then I have modified it, please help to review again, thanks.